src/http_server.cc

```diff
@@ -3233,8 +3237,11 @@ HTTPAPIServer::HandleGenerate(
 
   // If tracing is enabled see if this request should be traced.
   TRITONSERVER_InferenceTrace* triton_trace = nullptr;
-  std::shared_ptr<TraceManager::Trace> trace =
-      StartTrace(req, model_name, &triton_trace);
+  std::shared_ptr<TraceManager::Trace> trace;
+  if (trace_manager_) {
```

Comment:
do we want to test http frontend to make sure there are no segfaults? or this test was added in previous PRs?

Reply:
Just added a test case for http generate as well.